docs/discord-social-sdk/development-guides/sending-direct-messages.mdx

```diff
@@ -104,6 +104,21 @@ You can use this metadata to render a placeholder message for players and can li
 
 There is also more information about [messages in the Discord API](/docs/resources/message) documentation.
 
+## In-Game Direct Message Settings
+
+The Discord client provides a preference screen for users to be able to control who can DM them in-game via the
```

Comment:
minor nit:  "preference screen" sounds weird.  maybe "settings screen" instead?

Reply:
I like it! I'll make the change.